test/e2e/autoscale_test.go

```diff
@@ -277,6 +277,54 @@ func numberOfPods(ctx *testContext) (float64, error) {
 	return float64(resources.ReadyAddressCount(eps)), nil
 }
 
+func checkPodScale(ctx *testContext, targetPods, minPods, maxPods float64,  duration time.Duration) error {
+	// Short-circuit traffic generation once we exit from the check logic.
+	done := time.After(duration)
+	timer := time.Tick(2 * time.Second)
+
+	for {
+		select {
+		case <-timer:
+			// Each 2 second, check that the number of pods is at least `minPods`. `minPods` is increasing
```

Comment:
```suggestion
			// Every 2 seconds, check that the number of pods is at least `minPods`. `minPods` is increasing
```

Reply:
Second sentence doesn't compile for me gramatically, I'd reformulate it.

Reply:
I moved that sentence near the relevant code section